Tools/HolocronToolset/src/toolset/gui/editors/nss.py

```diff
@@ -183,14 +183,12 @@ def build(self) -> tuple[bytes | None, bytes]:
         if self._restype != ResourceType.NCS:
             return self.ui.codeEdit.toPlainText().encode("windows-1252"), b""
 
-        compiled_bytes: bytes | None = compileScript(self.ui.codeEdit.toPlainText(), self._installation.tsl, self._installation.path())
         print("compiling script from nsseditor")
+        compiled_bytes: bytes | None = compileScript(self.ui.codeEdit.toPlainText(), self._installation.tsl, self._installation.path())
         if compiled_bytes is None:
             print("user cancelled the compilation")
             return None, b""
-
-        msg = "Could not convert to bytes - nsseditor.build()"
-        raise ValueError(msg)
+        return compiled_bytes, b""
```

Comment:
Not sure how to do stacked changes in github: this commit is also in https://github.com/NickHugi/PyKotor/pull/72 which should be committed first.